ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -115,31 +114,38 @@ private static View findTouchTargetView(float[] eventCoords, ViewGroup viewGroup
     int childrenCount = viewGroup.getChildCount();
     // Consider z-index when determining the touch target.
     ReactZIndexedViewGroup zIndexedViewGroup =
-        viewGroup instanceof ReactZIndexedViewGroup ? (ReactZIndexedViewGroup) viewGroup : null;
+            viewGroup instanceof ReactZIndexedViewGroup ? (ReactZIndexedViewGroup) viewGroup : null;
+    String overflow = viewGroup instanceof ReactClippingViewGroup
+            ? ((ReactClippingViewGroup) viewGroup).getOverflow()
+            : null;
+    boolean allowOverflow = overflow == null || overflow.equals(ViewProps.VISIBLE);
+
     for (int i = childrenCount - 1; i >= 0; i--) {
       int childIndex =
           zIndexedViewGroup != null ? zIndexedViewGroup.getZIndexMappedChildIndex(i) : i;
       View child = viewGroup.getChildAt(childIndex);
-      PointF childPoint = mTempPoint;
+      TouchTestResult touchTestData = mTouchTestData;
+
       if (isTransformedTouchPointInView(
-          eventCoords[0], eventCoords[1], viewGroup, child, childPoint)) {
+          eventCoords[0], eventCoords[1], viewGroup, child, touchTestData, allowOverflow)) {
         // If it is contained within the child View, the childPoint value will contain the view
         // coordinates relative to the child
         // We need to store the existing X,Y for the viewGroup away as it is possible this child
         // will not actually be the target and so we restore them if not
         float restoreX = eventCoords[0];
         float restoreY = eventCoords[1];
-        eventCoords[0] = childPoint.x;
-        eventCoords[1] = childPoint.y;
-        View targetView = findTouchTargetViewWithPointerEvents(eventCoords, child);
+        eventCoords[0] = touchTestData.x;
```

Comment:
```suggestion
        eventCoords[0] = mTouchTestData.x;
```

Reply:
There was a tiny bug in the overflow test.
If you have change suggestions, please re-review them on the updated version.